internal/access/identity.go

```diff
@@ -86,35 +87,34 @@ func ListIdentities(c *gin.Context, opts data.ListIdentityOptions) ([]models.Ide
 	return data.ListIdentities(db, opts)
 }
 
+var ErrUserSyncFailed = fmt.Errorf("sync user failed") // this means either their session expired or they were removed from the idp
```

Comment:
Move to `internal/errors.go`?

Reply:
I think we should avoid putting more things into `internal/errors.go`. Application-wide errors are easy to mess up.  This error is specific to this one function, so defining it right next to that function is the right place for it, if we need it.

Do we need this to be an error variable? I can't find any code that checks this with `errors.Is`. Is that missing, or do you expect to need to need that later? If we aren't checking the error with `errors.Is`, I don't think we need a variable for it. We can inline it to the one place it is returned.

Reply:
Makes sense! Thanks!

Reply:
I meant to use this error to check that the error was expected as a sync fail, and if not log it at the error level rather than the debug level. I've made that change now.